app/views/application/_header.html.erb

```diff
@@ -33,4 +33,5 @@
       </ul>
     </div>
   </nav>
+  <%= render 'leave_site' %>
```

Comment:
Could move this to the `application.html.erb` layout since this button is not tied to the header?